cirq-core/cirq/testing/__init__.py

```diff
@@ -14,10 +14,6 @@
 
 """Utilities for testing code."""
 
-from cirq.testing.asynchronous import (
-    asyncio_pending,
```

Comment:
This is technically a breaking change. I suspect no one externally has been using our async testing helper code? but we should make a note of this for the release

Reply:
Brought this back since it's needed to test the work pool, which is now deprecated but not removed.